src/Bootstrappers/DatabaseTenancyBootstrapper.php

```
@@ -41,10 +69,44 @@ public function bootstrap(Tenant $tenant): void
+        // Check that no other tenant uses this tenant's database
+        if ($tenant::where($tenant->getTenantKeyName(), '!=', $tenant->getTenantKey())
+            ->where($tenant::getDataColumn() . '->' . $tenant->internalPrefix() . 'db_name', $tenantDbName)
+            ->exists()) {
+            throw new RuntimeException('Tenant cannot use a database of another tenant.');
+        }
+
+        $centralDbName = DB::connection(
+            config('tenancy.database.central_connection', 'central')
+        )->getDatabaseName();
+
+        if (DB::getDatabaseName() === $centralDbName) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'verifyTenantCanUseDatabase'. (undefined)

(StaticAccess)